app/src/main/java/com/postpc/tenq/ui/activities/ExistingRoomsActivity.java

```diff
@@ -32,7 +40,8 @@
 import retrofit2.Response;
 
 public class ExistingRoomsActivity extends TenQActivity {
-
+    private final MutableLiveData<ViewModelError> error = new MutableLiveData<>(null);
```

Comment:
Please move these and all network communication to a viewmodel class, like in JoinActivityViewModel